<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuentDialog</class>
 <widget class="QDialog" name="StuentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StuentDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="valueBox">
         <item>
          <property name="text">
           <string>学号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>成绩</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortBox">
         <item>
          <property name="text">
           <string>升序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>降序</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sortButton">
         <property name="text">
          <string>排序</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>学  员  ID：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2" colspan="2">
        <widget class="QLineEdit" name="idEdit"/>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>学员姓名：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>学员成绩：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="scoreEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="insertButton">
         <property name="text">
          <string>插入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectButton">
         <property name="text">
          <string>查询</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
